pkg/component/operator/image/v0/config/tasks.json

```diff
@@ -241,32 +526,19 @@
           "title": "Image",
           "type": "string"
         },
-        "objects": {
-          "$ref": "https://raw.githubusercontent.com/instill-ai/component/467caa4c05cf75d88e2036555529ecf6aa163b5c/resources/schemas/schema.json#/$defs/instill-types/keypoint/properties/objects",
+        "stuffs": {
+          "$ref": "https://raw.githubusercontent.com/instill-ai/component/467caa4c05cf75d88e2036555529ecf6aa163b5c/resources/schemas/schema.json#/$defs/instill-types/semantic-segmentation/properties/stuffs",
           "instillAcceptFormats": [
-            "array:structured/keypoint-object"
+            "array:structured/semantic-segmentation-stuff"
           ],
           "instillUpstreamTypes": [
             "reference"
           ]
-        },
-        "showScore": {
-          "description": "Show model confidence score on each instance",
-          "instillAcceptFormats": [
-            "boolean"
-          ],
-          "instillUIOrder": 0,
-          "instillUpstreamTypes": [
-            "value",
-            "reference"
-          ],
-          "title": "Show Score",
-          "type": "boolean"
         }
       },
       "required": [
         "image",
-        "objects"
+        "stuffs"
```

Comment:
Exactly, `stuff` is a term used in Panoptic Segmentation computer vision task (which is [different](https://encord.com/blog/panoptic-segmentation-guide) from Semantic Segmentation and Instance Segmentation task) for uncountable items. [COCO](https://doc.bdd100k.com/format.html) dataset has used the term since the beginning and it's a common label name now in the literature. We needs it to be plural form in this case because the field is literally an array of stuff object.